### ISSUE TITLE
hello

### ISSUE THREAD

**Issue by gurice** [Zylence/10-fast-fingers-typing-bot-lightning_fingers#2]
hello how are you sir? I'm the person you talked to 2 days ago

**Comment by Zylence** [comment]
Hey, what's up? Did you get your adaptions going?

**Comment by gurice** [comment]
Yes, I completed it, it works very well. But I have a little problem. I adapted this application to 3 different sites and it works very well on all of them, but I couldn't do whatever I did on one of them.

**Comment by gurice** [comment]
I am making the necessary arrangements, but it only writes 1 word and does not continue.

**Comment by gurice** [comment]
I will send you a video to help you understand this.

**Comment by gurice** [comment]
https://streamable.com/jk164g

**Comment by gurice** [comment]
Program only writes 1 word and leaves it, I can't figure it out, how do you think I should do it?

**Comment by gurice** [comment]
![screen](https://github.com/user-attachments/assets/2114f05d-11ec-4427-acef-7cc2032da83a)
The codes are like this

**Comment by gurice** [comment]
![24](https://github.com/user-attachments/assets/a2341ffe-aceb-4e6a-9296-d9de3c67abdc)

**Comment by gurice** [comment]
Okay, I solved it, I thank you again and again, I would like to ask you to please delete the 2 topics I opened, I don't want anyone to see it in the future.

**Comment by Zylence** [comment]
Hey, sorry for taking so long to check back with you. Sorry for not providing help, but GJ adjusting it. 💪

**Comment by Zylence** [comment]
> I would like to ask you to please delete the 2 topics I opened, I don't want anyone to see it in the future.

We can delete this one if you like, but I'd like to keep the other one, as it contributed to the repositories current state by requesting the sources to be published. Would this be fine with you?
